ksql-api/src/main/java/io/confluent/ksql/api/impl/Utils.java

```diff
@@ -49,53 +48,19 @@ private Utils() {
   }
 
   public static void checkIsWorker() {
-    checkThread(true);
-  }
-
-  public static void checkIsNotWorker() {
-    checkThread(false);
-  }
-
-  public static boolean isEventLoopThread() {
-    return isWorkerThread(false);
-  }
-
-  public static boolean isWorkerThread() {
-    return isWorkerThread(true);
-  }
-
-  private static boolean isWorkerThread(final boolean worker) {
-    final Thread thread = Thread.currentThread();
-    if (!(thread instanceof VertxThread)) {
-      throw new IllegalStateException("Not a Vert.x thread " + thread);
-    }
-    final VertxThread vertxThread = (VertxThread) thread;
-    return vertxThread.isWorker() == worker;
-  }
-
-  private static void checkThread(final boolean worker) {
-    if (!isWorkerThread(worker)) {
-      throw new IllegalStateException("Not a " + (worker ? "worker" : "event loop") + " thread");
```

Comment:
Not sure how this change is related to the subject of this PR.  Should this be in this PR?

Reply:
It's from the previously stacked PR.